R/code-of-conduct.R

```diff
@@ -16,7 +16,7 @@ use_code_of_conduct <- function(path = NULL) {
   if (!is.null(path)) {
     use_directory(path, ignore = is_package())
   }
-  save_as <- fs::path_join(c(path, "CODE_OF_CONDUCT.md"))
+  save_as <- path_join(c(path, "CODE_OF_CONDUCT.md"))
```

Comment:
Should this be just `path(path, "CODE_OF_CONDUCT.md")`?

Reply:
No, it has to be this way to gracefully handle the case where `path = NULL`.

``` r
library(fs)

path <- ".github"
path_join(c(path, "CODE_OF_CONDUCT.md"))
#> .github/CODE_OF_CONDUCT.md
path(path, "CODE_OF_CONDUCT.md")
#> .github/CODE_OF_CONDUCT.md

path <- NULL
path_join(c(path, "CODE_OF_CONDUCT.md"))
#> CODE_OF_CONDUCT.md
path(path, "CODE_OF_CONDUCT.md")
#> character(0)
```

Created on 2018-06-18 by the [reprex package](http://reprex.tidyverse.org) (v0.2.0).